var/spack/repos/builtin/packages/py-devito/package.py

```diff
@@ -16,51 +16,51 @@ class PyDevito(PythonPackage):
     """
 
     homepage = "https://www.devitoproject.org/"
-    pypi = "devito/devito-4.6.2.tar.gz"
+    pypi = "devito/devito-4.8.1.tar.gz"
 
-    version("4.6.2", sha256="39c2210a192ad69953b4f8d93440ffd72b07d739c4fe2290e2b182adfb7e143f")
+    version("4.8.1", sha256="56d0957a3226ed2a81c408107a614f04faa896d42c83a8b2bd1c8b1100adf51d")
 
     variant("mpi", default=False, description="Enable MPI support")
     variant("matplotlib", default=False, description="Enable matplolib support")
-    variant("ipyparallel", default=False, description="Enable ipyparallel support")
     variant("pandas", default=False, description="Enable pandas support")
 
     depends_on("py-pip@9.0.1:", type="build")
     depends_on("py-setuptools", type="build")
 
     depends_on("py-numpy@1.16:", type=("build", "run"))
-    depends_on("py-sympy@1.7:1.9", type=("build", "run"))
+    depends_on("py-sympy@1.9:1.12", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-sympy@1.9:1.11", type=("build", "run"))
```
Version ranges are inclusive